docs/installation/server-config/tuning-ksql.rst

```diff
@@ -0,0 +1,18 @@
+.. _tuning_ksql:
+
+When deploying KSQL to production, we recommend adding the following settings to your ``/etc/ksql/ksql-server.properties`` file:
```

Comment:
should avoid using `we` and instead write as: `When deploying KSQL to production, the following settings are recommended in your ``/etc/ksql/ksql-server.properties`` file:`

Reply:
Thanks, updated.